### PR TITLE
Upgrade node to 16

### PR DESCRIPTION
**Pull request by Zangetsu101**
Farajaland PR: https://github.com/opencrvs/opencrvs-farajaland/pull/552

### CODE REVIEW

File: packages/client/src/profile/serviceApi.ts [hunk 1]
```diff
@@ -54,7 +53,7 @@ function request<T>(options: AxiosRequestConfig) {
 
 const sendVerifyCode = (data: ISendVerifyCodeData) => {
   return request<ISendVerifyCodeResponse>({
-    url: resolve(window.config.API_GATEWAY_URL, 'sendVerifyCode'),
+    url: new URL(window.config.API_GATEWAY_URL, 'sendVerifyCode').toString(),
```
Comment (naftis) [inline packages/client/src/profile/serviceApi.ts:56]: The parameters should be other way around. With `new URL` the first one is the path, and the second one is the base URL.

File: packages/client/src/utils/apolloClient.ts [hunk 2]
```diff
@@ -39,7 +38,7 @@ export const createClient = (
   restoredCache?: InMemoryCache
 ) => {
   const httpLink = createHttpLink({
-    uri: resolve(window.config.API_GATEWAY_URL, 'graphql')
+    uri: new URL('graphql', window.config.API_GATEWAY_URL).toString()
```
Comment (naftis) [inline packages/client/src/utils/apolloClient.ts:41]: The parameters should be other way around. With new URL the first one is the path, and the second one is the base URL.

**Review by naftis — CHANGES_REQUESTED** [review]
minor change but otherwise LGTM

File: packages/client/src/profile/serviceApi.ts (hunk 1 above)
Comment (Zangetsu101) [reply packages/client/src/profile/serviceApi.ts:56]: Thanks Pyry! I must have missed this one

Comment (Zangetsu101) [inline packages/client/src/profile/serviceApi.ts:56]:
```suggestion
    url: new URL( 'sendVerifyCode', window.config.API_GATEWAY_URL).toString(),
```

**Review by euanmillar — APPROVED** [review]

**Pull request merged**